Assets/MixedRealityToolkit.Providers/Experimental/WindowsMixedReality/WindowsMixedRealitySpatialAwarenessSceneUnderstandingObserver.cs

```diff
@@ -0,0 +1,1245 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Timers;
+
+using UnityEngine;
+using UnityEngine.Assertions;
+using Microsoft.MixedReality.SceneUnderstanding;
+using Microsoft.MixedReality.Toolkit.Experimental.SpatialAwareness;
+using System.Threading;
+using System.Collections.Concurrent;
+
+using Microsoft.Windows.Perception.Spatial.Preview;
+using Microsoft.Windows.Perception.Spatial;
+
+#if WINDOWS_UWP
+using Windows.Storage;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Experimental.SpatialAwareness
+{
+    [MixedRealityDataProvider(
+        typeof(IMixedRealitySpatialAwarenessSystem),
+        SupportedPlatforms.WindowsUniversal,
+        "Windows Mixed Reality Scene Understanding Observer",
+        "Experimental/Profiles/DefaultSceneUnderstandingObserverProfile.asset",
+        "MixedRealityToolkit.SDK")]
+    public class WindowsMixedRealitySpatialAwarenessSceneUnderstandingObserver :
```

Comment:
Can you add a comment to the class name describing what it is and what it does, how it works, etc

Reply:
Nit: to PR authors and others, @keveleigh @davidkline-ms  @julenka  etc, what do people think about simplifying
WindowsMixedRealitySpatialAwarenessSceneUnderstandingObserver
to
WMRSpatialAwarenessSceneUnderstandingObserver ? 

I know other components outline "WindowsMixedReality" but this is a super long class name